lake_envs.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
```

Comment:
FastForest doesn't seem to specify the encoding at the top of the file, so I believe you can remove this